### DISCUSSION TITLE
레이아웃을 만들면서 도움이 된 CSS 참고 자료

### DISCUSSION THREAD

**Discussion by Leekyeongmi** [codestates/siphoon-note#72] [General]
일기 페이지 레이아웃을 만들면서 관련 CSS 속성을 공부하고 있는데, 이해하기 쉽게 정리가 잘 된 참고자료를 발견해 공유합니다! 

- Grid
https://studiomeal.com/archives/533
- Flex
https://studiomeal.com/archives/197

**[현재 진행 상황]**
- 일기 작성하는 창의 position 속성을 fixed로 만들어서, 오른쪽 창에서 무한스크롤을 하는 동안 왼쪽 창이 고정되도록 구현 
- 오른쪽 출력창은 현재 Flex 속성을 이용해서 카드를 두 개의 칼럼으로 분할하는 것까지는 성공을 했는데, 메세지 길이에 따라서 박스의 크기가 유동적으로 늘어나는 부분 구현중 (1차원 시스템인 Flex 달리 Grid는 가로/세로 2차원 레이아웃 조절이 가능해서 Grid를 이용해 다시 만들어 보려고 합니다.)

**[문제 공유]**
- 무한스크롤을 구현하게 될 경우 일기 분석하는 3개의 컨텐트 박스의 위치가 달라져야 할 가능성이 있음 (서버와 통신하는 방식의 변화는 없도록 고민 중)

**Comment by gomarag** [comment]
- 애니메이션 관련 툴
저도 검색하다가 봤는데 **크롬 개발자 도구에서 애니메이션 적용을 조절하고 직접 볼 수 있는 툴**인 것 같아서 링크 남겨봐요
[🔗개발자 도구의 모션 툴](https://motion.dev/tools)

> 처음 접속했을 때의 **랜딩페이지**에서 툴 사용법을 직관적으로 잘 보여주고 있는 것 같아서 랜딩페이지에 들어갈 내용에 도움도 될 것 같았어요😃

**Comment by gomarag** [comment]
> 무한스크롤을 구현하게 될 경우 일기 분석하는 3개의 컨텐트 박스의 위치가 달라져야 할 가능성이 있음 (**서버와 통신하는 방식의 변화는 없도록 고민 중**)

🔼이 부분은 아직 서버의 컨트롤러와 DB 테이블이 픽스되지 않아서 필요한 부분이 있다면 서버가 추가적으로 변경돼도 괜찮아요! 오후 미팅에서 좀 더 얘기해보면 좋겠네요!

**Reply by Leekyeongmi** [reply]
네! 그럼 이따 오후 회의에서 이야기해보도록 해요! :)
